libs/hdf-converters/src/netsparker-mapper.ts

```diff
@@ -201,25 +196,55 @@ export class NetsparkerMapper extends BaseConverter {
                 label: 'fix'
               }
             ],
-            refs: [],
-            source_location: {},
-            code: '',
+            impact: {
+              path: 'severity',
+              transformer: impactMapping(IMPACT_MAPPING)
+            },
+            code: {
+              transformer: (vulnerability: Record<string, unknown>): string =>
+                JSON.stringify(vulnerability, null, 2)
+            },
             results: [
               {
                 status: ExecJSON.ControlResultStatus.Failed,
                 code_desc: {path: 'http-request', transformer: formatCodeDesc},
                 message: {path: 'http-response', transformer: formatMessage},
-                run_time: 0,
                 start_time: {path: '$.netsparker-enterprise.target.initiated'}
               }
             ]
           }
         ],
         sha256: ''
       }
-    ]
+    ],
+    passthrough: {
+      transformer: (data: Record<string, unknown>): Record<string, unknown> => {
+        const auxData = _.get(data, 'netsparker-enterprise');
+        const genData = _.get(auxData, 'generated');
+        const targetData = _.omit(_.get(auxData, 'target'), [
```

Comment:
Could potentially just consolidate this to `_.get(data, 'netsparker-enterprise.generated')` and `_.get(data, 'netsparker-enterprise.target'`, but I feel like that might just be personal preference.